.eslintrc.js

```diff
@@ -1,28 +1,27 @@
 module.exports = {
+  root: true,
+
   env: {
-    'browser': true,
-  },
-  parserOptions: {
-    'ecmaVersion': 2018,
+    browser: true,
   },
+
   globals: {
-    'ethereum': 'readonly',
+    ethereum: 'readonly',
   },
-  plugins: [
-    'json',
-  ],
-  extends: [
-    '@metamask/eslint-config',
-    '@metamask/eslint-config/config/nodejs',
-  ],
-  overrides: [{
-    'files': ['src/*.js'],
-    'parserOptions': {
-      'sourceType': 'module',
+
+  extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
+
+  overrides: [
+    {
+      files: ['src/*.js'],
+      parserOptions: {
+        sourceType: 'module',
+      },
+      rules: {
+        camelcase: ['error'],
```

Comment:
:eyes: 

Why is this listed? The default camelcase rule should work fine here.

Reply:
Doh, I had overrides that I removed, and forgot to remove the rule itself. Fixed in 8683953